composer.json

```diff
@@ -31,19 +30,22 @@
         "twig/twig": "^2.14",
         "webmozart/assert": "^1.10"
     },
+    "conflict": {
+        "doctrine/doctrine-bundle": "2.3.0",
+        "symfony/doctrine-bridge": "4.4.16"
```

Comment:
Do you remember what was wrong with these? Because I don't remember having problems with them on other plugins